src/components/Dashboard/Dashboard.jsx

```diff
@@ -19,6 +27,7 @@ export class Dashboard extends React.Component {
                                         this.props.playedCard !== 0));
         return (
             <div className={styles.dashboard}>
+                <button data-cy="create-room" onClick={this.createRoom}>Create Room</button>
```

Comment:
This can use the Button component I think

Reply:
As in make a CreateRoom component that wraps a Button component with the createRoom function

Reply:
I'm not worried about styling yet, the PR is just for back-end functionality

Reply:
It's less for the styling and more so that we don't clutter up Dashboard with a bunch of functions

Reply:
Yup, I get what you mean, I'll change it when working on the front-end story